frontend/css/style.css

```diff
@@ -27,3 +26,24 @@ h2,
 .usa-button {
   width: 100%;
 }
+
+.loader {
```

Comment:
@natalialuzuriaga you should consider sending this spinner as an upstream contribution to USWDS!

Reply:
Solid idea 💡